docs/blog/2019-08-13-Localised-strings-from-Kentico-Cloud-GatsbyJS-GraphQL/index.md

```diff
@@ -0,0 +1,96 @@
+
+## Using GatsbyJS and GraphQL to retrieve the localised content
+
+Now we have the localised string Content Types and Content Items created in Kentico Cloud, the next step is to retrieve the content. I'm using the awesome and very popular static site generator [GatsbyJS](/) with [GraphQL](https://graphql.org/), a query language for your API.
+
+*(Another assumption for you... We’re assuming you have GatsbyJS + GraphQL set up for your project. If not, you can find a starting point here, [GatsbyJS sourcing from Kentico Cloud](/docs/sourcing-from-kentico-cloud/).)*
+
+With your content items in Kentico Cloud, you should be able to see them when you conduct a GraphQL query.
+
+Here is an example of the GraphQL query you will use to retrieve the ID and CodeName for the Content Items from Kentico Cloud using GatsbyJS. Think of this as the 'key' you would need to retrieve the 'value'. You can see the 'codename' and 'id' as potential options that you could use in the below GraphQL query.
+![A view of the GraphiQL preview of retrieving language nodes](images/GraphiQL-retrieve-lang-nodes.png)
+
+You can retrieve all localisation strings using the following GraphQL query.
+![Retrieving all language versions of the localised strings](images/GraphiQL-retrieve-lang-variants.png)
+
+However, to retrieve the actual 'value', there’s a little more to do in your GraphQL query.
+
+Here is an example of a GraphQL query where I am using the CodeName.
+![GraphiQL retrieve language nodes localised strings condition codename](images/GraphiQL-retrieve-lang-variants-based-on-condition-codename.png)
+
+And, here is an example of a GraphQL query where I am using the ID.
+![GraphiQL retrieve language nodes localised strings condition ID](images/GraphiQL-retrieve-lang-variants-based-on-condition-id.png)
+
+As you can see, once you know which 'key' you need, then it's pretty simple to get the 'value' and also detect the language variant you need to retrieve in GatsbyJS using this GraphQL query format.
+
+## Separate projects
+
+Before we go any further, it is important to take stock of your localisation string content items. Creating culture versions of each item doesn’t increase your number of Content Items so you’re unlikely to be jumping up through plans. However, the number of localisation string Content Items is something to keep in mind. Also, consider the number of environments/projects you have in your subscription, which could multiple the number of Content Items you have in total.
+
+We would always advocate pragmatism when creating your content models. In addition, maybe consider creating the localisation strings as a separate project that feeds into all the projects that need to utilise them. This is a solution that has worked across several of our projects and ensures that all localised strings are available to all projects and all environments, removing the overhead for updates.
```

Comment:
```suggestion
We would always advocate pragmatism when creating your content models. In addition, maybe consider creating the localization strings as a separate project that feeds into all the projects that need to utilize them. This is a solution that has worked across several of our projects and ensures that all localized strings are available to all projects and all environments, removing the overhead for updates.
```

Maybe find and replace other places where utilize pops up?

Reply:
yep, that's the best way and hopefully, I've now done this 👍